### PR TITLE
i18n(es): updates `internationalization`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description (required)

updates `internationalization` based on this commit 226c52579e2716553933ca78990b3b864987b8dd

#### Related issues & labels (optional)

- Closes # N/A
- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/es/guides/internationalization.mdx [hunk 1]
```diff
@@ -306,65 +307,57 @@ export default defineConfig({
...
-Con la configuración anterior:
-  
-- El archivo /fr/about.astro creará la URL `https://fr.example.com/about`.
-- El archivo /es/about.astro creará la URL `https://example.es/about`.
-- El archivo /ja/about.astro creará la URL `https://example.com/ja/about`.
-- El archivo /en/about.astro creará la URL `https://example.com/about`.
+#### Limitaciones
 
-Las URL anteriores también serán devueltas por las funciones `getAbsoluteLocaleUrl()` y `getAbsoluteLocaleUrlList()`.
+Esta carácteristica tiene algunas restricciones:
```
Comment (Waxer59) [inline src/content/docs/es/guides/internationalization.mdx:338]:
```suggestion
Esta característica tiene algunas restricciones:
```

File: src/content/docs/es/guides/internationalization.mdx [hunk 2]
```diff
@@ -306,65 +307,57 @@ export default defineConfig({
...
+- La opción `site` es obligatoria.
+- La opción `output` debe estar configurada en `"server"`.
+- No puede haber páginas prerenderizadas individuales.
+- La carácteristica del adaptador [`functionPerRoute`](/es/reference/adapter-reference/#functionperroute) no es admitida.
```
Comment (Waxer59) [inline src/content/docs/es/guides/internationalization.mdx:342]:
```suggestion
- La característica del adaptador [`functionPerRoute`](/es/reference/adapter-reference/#functionperroute) no es admitida.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias!

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59 😁

**Review by yanthomasdev — APPROVED** [review]
LGTM! Thanks 🙌

**Pull request merged**